lib/elixir/lib/integer.ex

```diff
@@ -400,17 +400,23 @@ defmodule Integer do
       iex> Integer.gcd(10, 0)
       10
 
+      iex> Integer.gcd(7, 7)
+      7
+
       iex> Integer.gcd(0, 0)
       0
 
   """
+  @spec gcd(0, 0) :: 0
+  @spec gcd(int, int) :: int when int: pos_integer
```

Comment:
@lexmag I did push, but i forgot to enter my password :fearful:   . It should be there now.